listener/Dockerfile

```diff
@@ -67,11 +64,35 @@ RUN OPENSSL_VER=1.1.1d \
  && THREADS=8 \
  && make -j$THREADS \
  && make test \
- && make install -j$THREADS \
- && ldconfig \
- && ln -s /etc/ssl/certs/* /usr/local/ssl/certs/ \
- && cd .. \
- && rm -rf openssl-$OPENSSL_VER
+ && make install -j$THREADS
+
+#Build Avalon Listener intermediate docker image
+FROM base_image as build_image
+
+RUN apt-get update \
+ && apt-get install -y -q \
+    cmake \
+    make \
+    libprotobuf-dev \
+    python3-dev \
+    python3-pip \
+    swig \
+    software-properties-common
+
+
+ENV OPENSSL_VER=1.1.1d
+COPY --from=openssl_image /usr/local/ssl /usr/local/ssl
+COPY --from=openssl_image /usr/local/bin /usr/local/bin
+COPY --from=openssl_image /usr/local/include /usr/local/include
+COPY --from=openssl_image /usr/local/lib /usr/local/lib
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER.tar.gz /tmp/

```

Comment:
Do we need to copy openssl-$OPENSSL_VER.tar.gz here ?

Reply:
OOPs. Its not needed here. Its copy and paste mistake. Will delete it.